Gemfile.lock

```diff
@@ -107,7 +107,7 @@ GEM
     mimemagic (0.3.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
```

Comment:
Not really important but this is left over from the now scrapped "try to get Ruby Head working" work